src/app/components/user/UserWorkspaces/PaginatedUserWorkspaces.js

```diff
@@ -66,8 +65,7 @@ const UserWorkspacesComponent = ({
   };
 
   const onSuccess = (team) => {
-    const path = `/${team.slug}/all-items`;
-    browserHistory.push(path);
+    window.location.assign(`/${team.slug}/all-items`);
```

Comment:
Suggestion: Let's leave a comment mentioning why this hard reset is important, otherwise it might get inadvertently replaced with a `browserHistory.push` :sweat_smile:

Reply:
Great idea, @amoedoamorim! Done!